### DISCUSSION TITLE
Controls and custom-elements.json

### DISCUSSION THREAD

**Discussion by digital-h** [storybookjs/storybook#14009] [Help]
Hi,

I have an auto-generated `custom-elements.json`. With the SB5 knobs-plugin I was able to use this also for the Canvas section and to filter out the properties I wanted to have there.

Example:
```
<Meta
  title="my-component"
  decorators={[withKnobs, withWebComponentsKnobs]}
  parameters={{ component: 'my-component', customElements: { filterProperties:['forceSelection', 'disabled', 'label', 'selectedIndices', 'displayFields', 'items'] }  }}
/>
```

Is there any way to do this with the control-plugin, which is the recommended way for SB6?

**Comment by shilman** [comment]
Addon-controls consumes `custom-elements.json` and uses it to automatically generate controls

This PR adds filtering in 6.2: https://github.com/storybookjs/storybook/pull/13898
